NEWS.md

```diff
@@ -31,13 +31,24 @@ values based on user-defined pairing (#944)
 - `derive_var_analysis_ratio` - Derives a ratio variable based on user-supplied variables
 from a BDS dataset, e.g. ADLB. (#943)
 
+- New high order functions (#701):
+
+  - `restrict_derivation()` - Execute a derivation on a subset of the input dataset
+  - `slice_derivation()` - The input dataset is split into subsets (slices) and
```

Comment:
Think lione 37 should be `The input dataset is split into slices (subsets) and`

Reply:
Updated